### DISCUSSION TITLE
Can't specify a directory to upload files

### DISCUSSION THREAD

**Discussion by atenadadkhah** [spatie/laravel-medialibrary#3033] [Q&A]
Hi. 
I'm using Spatie media library with [Filament](https://filamentphp.com/) in my project. 
This is the piece of code that makes the file input on the page.

    Section::make('image')->schema([
            Forms\Components\SpatieMediaLibraryFileUpload::make('image')
            ->disk('cloudinary')
            ->required()
            ->image()
            ->maxFiles(1)
            ->directory('/Elite Shop/blog')
            ->label('')
            ->collection('articles')
                        
    ])->collapsible(),

Here Spatie  uploads files to the root folder of Cloudinary but I want it to upload on `Elite Shop/blog` folder I tried specifing a `root` key on `filesystems.php` config file but it didn't work. 

    'cloudinary' => [
            'driver'         => 'cloudinary',
            'root' => 'Elite Shop',
            'api_key'        => env('CLOUDINARY_API_KEY'),
            'api_secret'     => env('CLOUDINARY_API_SECRET'),
            'cloud_name'     => env('CLOUDINARY_CLOUD_NAME'),
            'secure'         => env('CLOUDINARY_SECURE', true),
            'url' => env('CLOUDINARY_URL') ,
            'throw' => false,
                        
    ],
What should I do?

**Comment by fredsal** [comment]
Maybe try without the whitespace on `Elite Shop`

**Reply by atenadadkhah** [reply]
This didn't work either.
